inductiva/structures/scenarios/plate_linear_elastic/bc_utils.py

```diff
@@ -0,0 +1,105 @@
+"""Utils to create the different types of boundary conditions."""
+
+from typing import Optional, List
+
+import json
+
+
+class Dirichlet:
```

Comment:
Name this as `DirichletBC`, to emphasize that it represents boundary conditions. Same for Neumann.